--- a/generic/tk.h	2013-04-19 14:01:30.000000000 -0400
+++ b/generic/tk.h	2013-04-19 14:16:13.000000000 -0400
@@ -68,10 +68,7 @@
 #ifndef RC_INVOKED

 #ifndef _XLIB_H
-#   include <X11/Xlib.h>
-#   ifdef MAC_OSX_TK
-#	include <X11/X.h>
-#   endif
+#   include <tkX11Xlib.h>
 #endif
 #ifdef __STDC__
 #   include <stddef.h>

--- a/unix/Makefile.in	2013-04-19 14:33:55.000000000 -0400
+++ b/unix/Makefile.in	2013-04-19 14:35:55.000000000 -0400
@@ -553,7 +553,7 @@
 # Currently private, eventually public
 TTK_HDRS = $(TTK_DIR)/ttkTheme.h $(TTK_DIR)/ttkDecls.h

-PUBLIC_HDRS = $(GENERIC_DIR)/tk.h $(GENERIC_DIR)/tkDecls.h \
+PUBLIC_HDRS = $(GENERIC_DIR)/tk.h $(GENERIC_DIR)/tkDecls.h $(GENERIC_DIR)/tkX11Xlib.h \
 	$(GENERIC_DIR)/tkPlatDecls.h $(@TK_WINDOWINGSYSTEM@_HDRS)

 # The private headers we want installed for install-private-headers
