--- a/autogen.sh	2016-11-28 22:17:28.000000000 +0100
+++ b/autogen.sh	2016-11-28 23:01:27.429855855 +0100
@@ -19,7 +19,7 @@
 # directory is set up as an SVN external.
 if test ! -e gtest; then
   echo "Google Test not present.  Fetching gtest-1.7.0 from the web..."
-  curl -O https://googletest.googlecode.com/files/gtest-1.7.0.zip
+  curl -O http://download.linino.org/sources/gtest-1.7.0.zip
   unzip -q gtest-1.7.0.zip
   rm gtest-1.7.0.zip
   mv gtest-1.7.0 gtest
