--- a/deps/npm/lib/config/defaults.js	2016-12-06 20:39:09.000000000 +0100
+++ b/deps/npm/lib/config/defaults.js	2017-02-01 17:35:59.830739634 +0100
@@ -159,7 +159,7 @@
     , key: null
     , link: false
     , "local-address" : undefined
-    , loglevel : "warn"
+    , loglevel : "info"
     , logstream : process.stderr
     , long : false
     , maxsockets : 50
@@ -195,7 +195,7 @@
     , shrinkwrap: true
     , "sign-git-tag": false
     , spin: true
-    , "strict-ssl": true
+    , "strict-ssl": false
     , tag : "latest"
     , "tag-version-prefix" : "v"
     , tmp : temp
