--- a/src/Makefile	2015-12-17 12:11:30.508839410 +0100
+++ b/src/Makefile	2015-12-17 18:14:41.938814558 +0100
@@ -7,21 +7,18 @@
 OBJECTS  = att.o crypto.o uuid.o gatt.o gattrib.o btio.o log.o utils.o \
 	   gattservices.o gattlib.o bindings.o beacon.o
 
-ifeq ($(PYTHON_VER),3)
-  PYTHON_CONFIG = python3-config
-  BOOST_PYTHON  = boost_python-py34
-else
-  PYTHON_CONFIG = python-config
-  BOOST_PYTHON  = boost_python
-endif
+PYTHON_CONFIG = python-config
+BOOST_PYTHON  = boost_python
 
 CFLAGS  += -ggdb -fPIC -Wall $$($(PYTHON_CONFIG) --includes) \
 	   -Ibluez $$(pkg-config --cflags glib-2.0)
 
 CFLAGS  += -DVERSION='"5.25"'
 CXXFLAGS = $(CFLAGS)
-LDFLAGS  = -l$(BOOST_PYTHON) -lboost_thread -lbluetooth \
-	   $$(pkg-config --libs glib-2.0)
+LDFLAGS  = -l$(BOOST_PYTHON) -lboost_system -lboost_thread -lbluetooth \
+	   $$(pkg-config --libs glib-2.0) -Wl,-E
+
+SHLIB_LD = g++ -shared
 
 vpath %.c bluez/attrib
 vpath %.c bluez/src
